.github/workflows/kustomize_checks.yaml

```diff
@@ -0,0 +1,26 @@
+name: Node Kustomize checker
+
+on:
+  workflow_call:
+    inputs:
+      ENV_DIRECTORY:
```

Comment:
maybe this can be more meaningful? kustomize_dir or directory(as its already a kustomize only workflow)?